tajo-docs/src/main/sphinx/jdbc_driver.rst

```diff
@@ -9,11 +9,15 @@ In this section, we explain how to get JDBC driver and an example code.
 How to get JDBC driver
 =======================
 
+From Tajo Homepage
```

Comment:
I have one trivial suggestion. How about 'Direct Download'? In my view, It would be better.